dnf-behave-tests/common/output.py

```diff
@@ -140,6 +140,6 @@ def then_stderr_is(context):
 
 @behave.then("stderr matches line by line")
 def then_stderr_matches_line_by_line(context):
-    out_lines = context.cmd_stderr.split('\n')
-    regexp_lines = context.text.split('\n')
+    out_lines = context.cmd_stderr.strip().split('\n')
+    regexp_lines = context.text.strip().split('\n')
```

Comment:
tbh. I'm a bit torn on the `strip()` here. Ultimately we'd like to check for the correct number of line breaks too. Once we put this in, we probably won't look back... but yeah, we're stripping the output for `stdout` already...

I assume you've had a particular reason to do this?

Reply:
I also think we probably should not strip either the stdout or stderrr. I did this because I was getting the output from stderr surrounded by two empty ones, but now that I think about it I just treated the symptoms.. I will further look into it.

Reply:
It turned out the empty new line really is in the output of createrepo_c, thanks for catching this.